llvm/utils/lit/lit/formats/standardlibrarytest.py

```diff
@@ -0,0 +1,355 @@
+# ===----------------------------------------------------------------------===##
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# ===----------------------------------------------------------------------===##
+
+import lit
+import os
+import re
+from .base import FileBasedTest
+
+
+def _getSubstitution(substitution, config):
```

Comment:
There is a duplicate of this function in `utils/libcxx/test/config.py`. Cleaning this up is on my todo list, however would it make sense to do that in this patch?

Reply:
Agreed, I think it makes sense to do that in this patch.